app/src/main/kotlin/io/homeassistant/companion/android/assist/ui/AssistSheetView.kt

```diff
@@ -81,8 +77,12 @@ import com.mikepenz.iconics.typeface.library.community.material.CommunityMateria
 import io.homeassistant.companion.android.R
 import io.homeassistant.companion.android.common.R as commonR
 import io.homeassistant.companion.android.common.assist.AssistViewModelBase
+import io.homeassistant.companion.android.util.compose.safeScreenHeight
 import kotlinx.coroutines.launch
 
+private val HEADER_HEIGHT = 48.dp
+private val CONTROL_HEIGHT = 112.dp
```

Comment:
There are multiple buttons, and let's use the same name as the composable

```suggestion
private val CONTROLS_HEIGHT = 112.dp
```